src/libs/OptionsListUtils.js

```diff
@@ -209,31 +205,37 @@ function isPersonalDetailsReady(personalDetails) {
 
 /**
  * Get the participant options for a report.
- * @param {Object} report
+ * @param {Array<Object>} participants
  * @param {Array<Object>} personalDetails
  * @returns {Array}
  */
-function getParticipantsOptions(report, personalDetails) {
-    const participants = lodashGet(report, 'participantAccountIDs', []);
-    return _.map(getPersonalDetailsForAccountIDs(participants, personalDetails), (details) => ({
-        keyForList: String(details.accountID),
-        login: details.login,
-        accountID: details.accountID,
-        text: details.displayName,
-        firstName: lodashGet(details, 'firstName', ''),
-        lastName: lodashGet(details, 'lastName', ''),
-        alternateText: Str.isSMSLogin(details.login || '') ? LocalePhoneNumber.formatPhoneNumber(details.login) : details.login || details.displayName,
-        icons: [
-            {
-                source: UserUtils.getAvatar(details.avatar, details.accountID),
-                name: details.login,
-                type: CONST.ICON_TYPE_AVATAR,
-                id: details.accountID,
-            },
-        ],
-        payPalMeAddress: lodashGet(details, 'payPalMeAddress', ''),
-        phoneNumber: lodashGet(details, 'phoneNumber', ''),
-    }));
+function getParticipantsOptions(participants, personalDetails) {
+    const details = getPersonalDetailsForAccountIDs(_.pluck(participants, 'accountID'), personalDetails);
+    return _.map(participants, (participant) => {
```

Comment:
Why did we have to change how this loop works?

Reply:
Because `participants` is now an array of objects, it looks like this: `[{accountID: 1, selected: true, ...}]`. Previously, it was a report object and we take the `participantAccountIDs` which is just an array.
<img width="606" alt="image" src="https://github.com/Expensify/App/assets/50919443/290df4c5-04f4-4710-8a9d-2df2e5aec779">

Also, we need `selected` attribute from each `participant`.

Reply:
Thanks